genie-web/src/main/java/com/netflix/genie/web/configs/grpc/GenieGRpcServicesAutoConfiguration.java

```diff
@@ -51,7 +49,6 @@
  * @since 4.0.0
  */
 @Configuration
-@ConditionalOnProperty(value = GRpcServerProperties.ENABLED_PROPERTY, havingValue = "true")
```

Comment:
Have you noticed any different in runtime for tests?
Any integration test that does not exclude this configuration now will bring up a gRPC server.
And i wonder if that makes any noticeable difference.

Reply:
Haven't noticed any noticeable difference but disabling it is still possible by providing a `noop` bean of type `GRpcServerManager` in the test context. Then it just won't start the server. But really it's a netty server listening on a port. Seems almost instant at least compared to tomcat.